tools/ld-chroma-decoder/main.cpp

```diff
@@ -152,6 +152,16 @@ int main(int argc, char *argv[])
                                         QCoreApplication::translate("main", "number"));
     parser.addOption(chromaGainOption);
 
+    // Option to output packed RGB (--rgb)
```

Comment:
It'd be simpler to just have one option -- either leave RGB as the default and have a --yuv flag to enable YUV output, or add an output format option that takes a string (like -f).

(If you have two boolean options, then you also need to check the user hasn't specified both of them below.)

Reply:
I thought I tested setting both, but maybe not or I didn't try reordering.  Anyway, the next patch will be to add y4m output (yuv with headers), so having a more general output-format option would be good.

I'd argue YCbCr should be the default, but I don't want to break everyone's scripts.

Reply:
Dropped `--rgb` and `--yuv`

Use `--output-format` and `-p` (for pixel?).